created template for merge request onto main (#13)
energinet-singularity/dlr-parser: .github/pull_request_template.md

> **Description of your changes:**

Reviewer: I perceive this line as a heading not a listing and therefore suggest you remove the colon :)
```suggestion
**Description of your changes**
```

Category:

* [ ] New Feature
* [ ] Bugfix

> * [ ] Documentation fixed

Reviewer: Just a wording thing.. :-P
```suggestion
* [ ] Documentation update
```

Checklist:

> * [ ] I have bumped the chart version

Reviewer: There might be versioning for the application and the chart - perhaps we would like to have the commit'er check both?
```suggestion
* [ ] I have bumped all relevant version numbers
```

* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have documented my changes if this is needed
* [ ] I have described my changes in the "description of your changes" field
* [ ] I have tested this change on a running cluster 

> * [ ] I have created the necessary tests in the chart, if they are possible/necessary

Reviewer: I am uncertain if misunderstood this line? What chart?
```suggestion
* [ ] I have created/corrected automated tests related to the changes, if possible/necessary
```

> * [ ] New and existing unit tests pass locally with my changes

Reviewer: Locally?

Author: Yes, think it is okay that people have tested pylint before PR.
